sdk/ml/azure-ai-ml/azure/ai/ml/_exception_helper.py

```diff
@@ -86,7 +86,7 @@ def format_details_section(
 
     if hasattr(error, "message"):
         error_types[error.error_type] = True
-        details += f"\n\n{error.message}\n"
+        details += f"\n\n{Fore.RED}{error.message}{Fore.RESET}\n"
```

Comment:
This is very cool. How does it look like in a jupyter notebook?

Reply:
![image](https://user-images.githubusercontent.com/16376603/205216823-18282873-b0ce-480c-9c4b-1e6bb3c2b101.png)

Reply:
![image](https://user-images.githubusercontent.com/16376603/205217620-c617431c-5fbb-428b-b363-9afc3404a309.png)